docs/sources/_index.md

```diff
@@ -31,20 +35,19 @@ Explore Logs is currently in [public preview](/docs/release-life-cycle/). Grafan
 
 # Explore Logs documentation
 
-## Welcome to our new logs experience
+![A new way to look at your logs](images/explore-logs-hero-banner.png)
+![Screenshot of Explore Logs landing page](images/explore-logs-features.jpeg)
```

Comment:
I think this screenshot is too busy for documentation.  It's the sort of thing I'd expect to see in a blog post or marketing,  not user documentation.